packages/drivers/odsp-driver-definitions/api-report/odsp-driver-definitions.api.md

```diff
@@ -235,7 +220,9 @@ export const OdspErrorTypes: {
     readonly fetchFailure: "fetchFailure";
     readonly fetchTokenError: "fetchTokenError";
     readonly incorrectServerResponse: "incorrectServerResponse";
-    readonly fileOverwrittenInStorage: "fileOverwrittenInStorage";
+    readonly fileOverwrittenInStorage: "fileOverwrittenInStorage"; /**
```

Comment:
For visibility, this is a known API-Extractor issue. So long as CI generates the same content, this isn't an issue.